src/index.html

```diff
@@ -16,17 +16,48 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
```

Comment:
HTML Content: Class names represent the meaning of the content (not the styles or tag names). It's a good practice to use meaningful class names that represent the content or purpose of the elements, rather than their appearance. This makes your code easier to read and maintain.